gateway/ln-gateway/src/gateway_module_v2/receive_sm.rs

```diff
@@ -58,6 +59,7 @@ pub struct ReceiveSMCommon {
     pub contract: IncomingContract,
     pub out_point: OutPoint,
     pub refund_keypair: Keypair,
+    pub amount: Amount,
```

Comment:
Also a breaking change that needs a migration, is it worth it?

Reply:
Thanks, good catch. Forgot to remove this, its not needed now.

Reply:
Removed